src/modules/organization/components/InternalGroupDetail/InternalGroupUserHighlights.tsx

```diff
@@ -1,9 +1,105 @@
+import {
+  ActionIcon,
+  Badge,
+  Card,
+  Group,
+  Image,
+  Modal,
+  SimpleGrid,
+  Spoiler,
+  Text,
+  Title,
+  useMantineTheme,
+} from '@mantine/core'
+import { useQuery } from '@apollo/client'
+import { INTERNAL_GROUP_USER_HIGHLIGHTS_BY_INTERNAL_GROUP_QUERY } from '../../queries'
+import { FullPageError } from 'components/FullPageComponents'
+import { FullContentLoader } from 'components/Loading'
+import {
+  InternalGroupUserHighlightNode,
+  InternalGroupUserHighlightsByInternalGroupReturns,
+  InternalGroupUserHighlightsByInternalGroupVariables,
+} from '../../types'
+import { InternalGroupUserHighlightEditForm } from 'modules/organization/components/InternalGroupDetail/components/EditHighlights/InternalGroupUserHighlightEditForm'
+import { useState } from 'react'
+import { IconNotes } from '@tabler/icons'
+
 interface InternalGroupUserHighlightsProps {
   internalGroupId: string
 }
 
 export const InternalGroupUserHighlights: React.FC<
   InternalGroupUserHighlightsProps
 > = ({ internalGroupId }) => {
-  return <span>Her kommer beskrivelser om noen folk i gjengen</span>
+  const { data, loading, error } = useQuery<
+    InternalGroupUserHighlightsByInternalGroupReturns,
+    InternalGroupUserHighlightsByInternalGroupVariables
+  >(INTERNAL_GROUP_USER_HIGHLIGHTS_BY_INTERNAL_GROUP_QUERY, {
+    variables: { internalGroupId: internalGroupId },
+  })
+  const [modalOpened, setModalOpened] = useState(false)
+  const [selectedHighlight, setSelectedHighlight] =
+    useState<InternalGroupUserHighlightNode | null>(null)
+  const theme = useMantineTheme()
+  if (error || !data) return <FullPageError />
+  if (loading) return <FullContentLoader />
```

Comment:
`!data` whould be moved to loading
`if (loading || !data)`

Reply:
fixed